--- a/py/mockbuild/plugins/hw_info.py	2019-11-01 15:06:54.000000000 +0100
+++ b/py/mockbuild/plugins/hw_info.py	2019-11-28 18:58:42.265710896 +0100
@@ -42,7 +42,7 @@
         out.write("CPU info:\n")
         out.write(output)
 
-        cmd = ["/bin/free"]
+        cmd = ["/usr/bin/free"]
         output = mockbuild.util.do(cmd, shell=False, returnOutput=True, raiseExc=False)
         out.write("\n\nMemory:\n")
         out.write(output)

