Author: Tyler Trafford <ttrafford@gmail.com>
Description: Adapt PAM setting to Debian
Forwarded: no

--- a/etc/pam/mock
+++ b/etc/pam/mock
@@ -5,9 +5,9 @@
 #auth		sufficient	pam_wheel.so trust use_uid
 # Uncomment the following line to require a user to be in the "wheel" group.
 #auth		required	pam_wheel.so use_uid
-auth		include		system-auth
+auth		include		common-auth
 account		sufficient	pam_succeed_if.so user ingroup mock use_uid quiet
-account		include		system-auth
-password	include		system-auth
-session		include		system-auth
+account		include		common-account
+password	include		common-password
+session		include		common-session
 session		optional	pam_xauth.so
